src/command-registry.coffee

```diff
@@ -93,6 +93,9 @@ class CommandRegistry
       return disposable
 
     if typeof target is 'string'
+      if typeof callback isnt 'function'
```

Comment:
I think this might need to go above line 95, it looks like it only throws when using the selector version of this API and not when using a target element directly.

It might be good to have specs for that case as well, throwing when doing the following:

```coffee
atom.commands.add(document.body, 'foo:command')
```